src/components/organisms/news/news.html

```diff
@@ -0,0 +1,49 @@
+<div class="news">
+  <div class="news-and-events-title">
+    <h2>News<span class="caption">お知らせ</span></h2>
+  </div>
+  <article class="news-contents-wrapper">
+    <a href="#" class="news-to-click">
+      <div class="news-content">
+        <time class="date" datetime="2018.06.20">2018.06.20</time>
+        <section class="news-title">
+          ブックカフェ・コワーキング系サイト向けテーマ「Switch」をリリースしました。
+        </section>
+      </div>
```

Comment:
```suggestion
      <article class="news-content">
        <time class="date" datetime="2018-06-20">2018.06.20</time>
        <h3 class="news-title">
          ブックカフェ・コワーキング系サイト向けテーマ「Switch」をリリースしました。
        </h3>
      </article>
```
timeタグが使えてていい感じです！細かい点になるのですが、datetime属性は書式が決まっているのでmdnで確認いただけると幸いです。

また、それぞれの記事をarticleタグで囲みたいのでこのようにするのが適切かと思います。
記事タイトルに当たる部分にはheadingタグを使いましょう。news-and-events-titleがh2タグなので、階層構造的にh3が妥当だと思います。


article: https://developer.mozilla.org/ja/docs/Web/HTML/Element/article
time: https://developer.mozilla.org/ja/docs/Web/HTML/Element/time
heading: https://developer.mozilla.org/ja/docs/Web/HTML/Element/Heading_Elements